core/githubhelper.py

```
@@ -331,6 +331,26 @@ def get_libraries_json(self, repo_slug: str, tag: str = "master"):
+
+        Many libraries might not ship this file, so it will returns None quietly if not found.
+        See libraries/website_adoc.py for the parser and the file's contract.
+
+        :param repo_slug: str, the repository slugs
+        :param tag: str, the Git tag
+        :return: bytes, the file content, or None if it doesn't exist
+        """
+        url = f"https://raw.githubusercontent.com/{self.owner}/{repo_slug}/{tag}/meta/website.adoc"  # noqa
+
+        try:
+            response = requests.get(url)
```

Ruff (0.15.20)

[error] Probable use of `requests` call without timeout

(S113)